Python/Random_Youtube_Videos/randomyt.py

```diff
@@ -0,0 +1,16 @@
+import string
+import random
+import webbrowser
+import requests
+
+try:
+    API_KEY = str(input("Enter your generated API-KEY : "))
```

Comment:
Pass the API key from a `.env` file

Reply:
Done through .env file... Also provided a sample .env_sample file